exporter/clickhousetracesexporter/writer.go

```diff
@@ -251,17 +244,34 @@ func (w *SpanWriter) writeIndexBatch(batchSpans []*Span) error {
 func (w *SpanWriter) writeTagBatch(batchSpans []*Span) error {
 
 	ctx := context.Background()
-	statement, err := w.db.PrepareBatch(ctx, fmt.Sprintf("INSERT INTO %s.%s", w.traceDatabase, w.attributeTable))
+	tagStatement, err := w.db.PrepareBatch(ctx, fmt.Sprintf("INSERT INTO %s.%s", w.traceDatabase, w.attributeTable))
 	if err != nil {
 		logBatch := batchSpans[:int(math.Min(10, float64(len(batchSpans))))]
 		w.logger.Error("Could not prepare batch for span attributes table: ", zap.Any("batch", logBatch), zap.Error(err))
 		return err
 	}
-
+	tagKeyStatement, err := w.db.PrepareBatch(ctx, fmt.Sprintf("INSERT INTO %s.%s", w.traceDatabase, w.attributeKeyTable))
+	if err != nil {
+		logBatch := batchSpans[:int(math.Min(10, float64(len(batchSpans))))]
+		w.logger.Error("Could not prepare batch for span attributes key table: ", zap.Any("batch", logBatch), zap.Error(err))
+		return err
```

Comment:
You are already logging in the caller - avoid logging again and just return the err up in the call chain.

Reply:
Yes